src/util/system.cpp

```diff
@@ -63,6 +63,7 @@
 #endif
 
 #include <thread>
+#include <univalue.h>
```

Comment:
Is this the right place to put this include? Shouldn't it be way more up, together with other bitcoin stuff?

Reply:
>  Univalue is technically an external include

Oh, right then.